content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -27,8 +27,8 @@ <h3>Objectives</h3>
                 <h3>Kubernetes Deployments</h3>
                 <p>
                 Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of it.
-                To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs Kubernetes
-                how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
+                To do so, you create a Kubernetes <b>Deployment</b> configuration (Kubenetes Object that describes the desired state of deployment). The Deployment instructs Kubernetes
```

Comment:
To do so, you create a Kubernetes <b>Deployment</b> (a Kubernetes object that declares your intent to
run a set of application instances). The Deployment instructs Kubernetes

Reply:
I am unable to add this suggestion to the batch. pls look into it.